server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -803,24 +803,38 @@ public Iterator<Setting<?>> settings() {
      * occupy at most 4 bytes.
      */
 
+    public static final int IGNORE_ABOVE_DEFAULT_STANDARD_INDICES = Integer.MAX_VALUE;
```

Comment:
Would it make sense to move these constants to `IgnoreAbove`?

Reply:
Maybe the default can be defined in the constants in `IgnoreAbove`? And add a constructor to `IgnoreAbove` that accepts value and defaultValue?

Reply:
This would avoid `IgnoreAbove` relying on `IndexSettings`?

Reply:
yeah I can move them.

Regarding having a constructor with `value` and `defaultValue`. I purposefully left out the `defaultValue` to reduce the odds of someone passing in the wrong one (Integer.MAX_VALUE instead of 8191 for logs). What do you think?

Reply:
Maybe that constructor should have private visibility? That way I can't be used by accident?